portfolio/src/main/webapp/contact.html

```diff
@@ -23,28 +25,33 @@
 
 	<div class="contact-me">
 		<h1>Contact Me!</h1>
-		<form action="/data" method="POST">
+		<form id="comment-form" method="POST" enctype="multipart/form-data">
 			<div class="form-content">
 				<label for="name">Name</label>
 				<input type="text" name="name" id="name" placeholder="optional"><br>
-            </div>
+      		</div>
+			<div class="form-content">
+				<label for="name">Upload an image</label>
+				<input type="file" name="image"><br>
+			</div>
 			<div class="form-content">
-                <label for="comment">Comments</label>
-                <textarea id="comment" name="comment" rows="7" cols="30" placeholder="Type your comments here!" required></textarea><br>
-            </div>
-            <button role="submit" name="commentButton">Submit</button>
+				<label for="comment">Comments</label>
+				<textarea id="comment" name="comment" rows="7" cols="30" placeholder="Type your comments here!" required></textarea><br>
+			</div>
+			<button role="submit" name="commentButton">Submit</button>
 		</form>
-        <form>
-            <div>
-                <label>Number of Comments: </label>
-                <input type="number" min="0" max="50" id="numComments" onchange="updateNumComments();">
-            </div>
-        </form>
-        <button onclick="deleteComments();">Delete Comments<button>
-	<fieldset id="user-comments">
+		<form>
+			<div>
+				<label>Number of Comments: </label>
+				<input type="number" min="0" max="50" id="numComments" onchange="updateNumComments();">
+      		</div>
+		</form>
```

Comment:
How does this form get submitted?

Reply:
> How does this form get submitted?

The form is not submit, I removed it. Everytime, a user enters a number, it is automatically registered